include/clang/Basic/DiagnosticGroups.td

```diff
@@ -391,6 +391,7 @@ def ObjCPointerIntrospectPerformSelector : DiagGroup<"deprecated-objc-pointer-in
 def ObjCPointerIntrospect : DiagGroup<"deprecated-objc-pointer-introspection", [ObjCPointerIntrospectPerformSelector]>;
 def ObjCMultipleMethodNames : DiagGroup<"objc-multiple-method-names">;
 def ObjCFlexibleArray : DiagGroup<"objc-flexible-array">;
+def Odr : DiagGroup<"odr">;
```

Comment:
Probably "OneDefinitionRule" is a better name? Not every user may know about the ODR abbrevation, and it would be not longer than other names.

Reply:
Good point, I have renamed the group but kept the option that is controlling whether the diagnostic is an error or warning as is (-Werror=odr). What do you think, should that also be named '-Werror=one-definition-rule'?